lxd/network/openvswitch/ovn.go

```diff
@@ -1085,6 +1085,40 @@ func (o *OVN) LogicalSwitchPorts(switchName OVNSwitch) (map[OVNSwitchPort]OVNSwi
 	return ports, nil
 }
 
+// LogicalSwitchPortIPs returns a list of IPs associated to each port connected to switch.
+func (o *OVN) LogicalSwitchPortIPs(switchName OVNSwitch) (map[OVNSwitchPort][]net.IP, error) {
+	output, err := o.nbctl("--format=csv", "--no-headings", "--data=bare", "--colum=name,addresses,dynamic_addresses", "find", "logical_switch_port",
+		fmt.Sprintf("external_ids:%s=%s", ovnExtIDLXDSwitch, switchName),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	// Parse lines like:
```

Comment:
Missing example?

Reply:
Good spot

Reply:
I took it out as the comment lower down suffices.